QuizAppNew/View/QuestionView.swift

```diff
@@ -9,10 +9,15 @@ import SwiftUI
 
 struct QuestionView: View {
     @Binding var isResultShow : Bool
+    //MARK: Crate variable for load json item
```

Comment:
why is below the binding variable declared?

Reply:
deleted binding property wrappers

Reply:
why deleted? why did you declare?

Reply:
@Binding variable read and write kore value tar shottota jachai kore.. isResultShow ei variable e 2 ta connection er maddhome data store kore. variable(Bool) jokhon true hoy token view true abar jokhon false hoy token view ta false hoy

Reply:
isResultShow delete kori ni